src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const visibleTabId =
+    tabs.findIndex(tab => tab.id === selectedTabId) > -1
+      ? selectedTabId
+      : tabs[0].id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => {
+              return (
+                <li
+                  key={tab.id}
+                  className={cn({
+                    'is-active': tab.id === visibleTabId,
```

Comment:
You can also create a variable inside map function to avoid code duplication

`const isTabSelected = tab.id === visibleTabId`